website/prebuild.py

```diff
@@ -84,17 +84,25 @@ def label_subdir(
     readme = subdir / "README.md"
     readme_content = readme.read_text()
 
+    def include_child(md_file: Path) -> bool:
+        """Determines whether a markdown file should be included in the
+        navigation tree as a child of this label.
```

Comment:
```suggestion
        """Returns true for direct children in the navigation tree.
```

(suggesting as the PEP8 single-line-first-line convention)

Reply:
```suggestion
        """Returns whether md_file is a child of this label.
```